packages/grid_client/scripts/tft.ts

```diff
@@ -0,0 +1,28 @@
+import { substrateURL, TFT } from "../src";
```

Comment:
Please check the other scripts and let's do the same here, we need to define a main function, separate each block into its function, and use a block logger

Reply:
Edited.